docs/configuration.md

```diff
@@ -4869,6 +4871,22 @@ literals.long=Upper
 0xaAaAl
 ```
 
+### `literals.binPrefix`
+
+```scala mdoc:defaults
+literals.binPrefix
+```
+
+Responsible for the case of binary integer literals prefix `0b`
```

Comment:
could you please prepend `(Since v3.8.4)`, to make it clear when this parameter became available?

Reply:
looks like you mentioned this aspect in your PR comment as well :)